website/docs/r/container_bind_service.html.markdown

```diff
@@ -8,6 +8,11 @@ description: |-
 ---
 
 # ibm_container_bind_service
+
+> [!CAUTION]
+> This resource is not reliable and will be deprecated.
+It is highly recommended to bind your services according to [this guide](I'm unsure how to link to the newly written guide)

```

Comment:
I will need some guidance here on how to link another file

Reply:
which link you mean here .Link outside this files?

Reply:
The new guide I've added, in `website/docs/guides/binding-services.html.md`

Reply:
Done in [b10b72a](https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5588/commits/b10b72a27457bcc8b838cf187399893789a6b500)